### DISCUSSION TITLE
Unable to log in if invalid jwt cookie is already set at parent domain

### DISCUSSION THREAD

**Discussion by Ivo-Donchev** [requarks/wiki#3269] [Error / Bug Report]
**Describe the bug**
If you host wikijs on a subdomain `wiki.domain.com` and the site you host on `domain.com` sets a jwt cookie (with `jwt` name), there's a conflict with the cookies when you try to log into wiki.js which doesn't allow to login but displays `Login successful message` 

**To Reproduce**
1. Host wikijs on subdomain (wiki.domain.com)
2. Open wiki.domain.com/login 
2. Open console in browser and type `document.cookie = "jwt=test; domain=domain.com"`
3. Enter your credentials and click `Log in`
4. See "Login successful! Redirecting..." which will redirect you back to the login since it looks the cookie from the upper domain 

**Expected behavior**
I think wiki js should keep looking the cookies from the upper domain but should look it's own with higher priority. Or to provide interface for configuring `jwt` cookie name to something custom  

**Host Info (please complete the following information):**
 - OS: Ubuntu 16.04, Docker
 - Wiki.js version: 2.5.144
 - Database engine: postgres 9.7

**Comment by KallistoX** [comment]
Same Problem here, have you found a solution how to log in again? I am locked out of my wiki right now. Can I set the jwt domain in a configuration file manually? Thankful for any help

**Reply by KallistoX** [reply]
solved by restarting the wiki service...

**Comment by yusufgungor** [comment]
Hi, this problem exist for us too. Can we make jwt cookie name to something custom?

**Comment by yusufgungor** [comment]
Hi @NGPixel what do you think about this issue? Is it hard to make jwt cookie name custom?
